src/ucp/core/ucp_am.h

```diff
@@ -73,16 +80,6 @@ typedef struct {
 } ucp_am_first_desc_t;
 
 
-typedef struct {
-    ucp_rndv_rts_hdr_t       super;
-    ucp_am_hdr_t             am;
-    /*
-     * 1. packed rkeys follow
-     * 2. user header follows, if am->header_length is not 0
-     */
-} UCS_S_PACKED ucp_am_rndv_rts_hdr_t;
```

Comment:
which "old" UCX version really need AM compatibility? do we miss any functionality after removing `ucp_am_hdr_t`?

Reply:
previous version doesn't have AM RNDV

Reply:
then why need to remove current?

Reply:
it is needed to use common RNDV header for TAG and AM.
This is needed to identify what type of message is received - AM or TAG